api/api/pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+asyncio_mode=auto
```

Comment:
Tried creating the file inside the `test` folder but it didn't prevent the warning from showing. Same happened in the `wodles` module.